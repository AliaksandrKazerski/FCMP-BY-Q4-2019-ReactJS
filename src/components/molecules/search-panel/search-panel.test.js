import React from 'react';
import { shallow } from 'enzyme';

import SearchPanel from './index';

jest.mock('../../../../img/netflix2.png', ()=>'mock.png');

describe('SearchPanel', () => {
  it("SearchPanel should render correctly", () => {
    const mockSearchParams = {
      search: 'mock',
      searchBy: 'mock',
      sortBy: 'mock',
    };

    const myComponent = shallow(
      <SearchPanel
        searchParams={mockSearchParams}
      />
    );
  
  expect(myComponent).toMatchSnapshot()
  });
});