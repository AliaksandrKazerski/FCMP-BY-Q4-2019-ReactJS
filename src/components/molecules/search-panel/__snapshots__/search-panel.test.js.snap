// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPanel SearchPanel should render correctly 1`] = `ShallowWrapper {}`;
